clang/test/Frontend/Inputs/dependency.cpp

```diff
@@ -0,0 +1,3 @@
+int main() {
+  return 0;
+}
```

Comment:
Looks like you are getting a ProtexIP hit on this code.  It's a simple main() function but maybe just create a more unique name for your function since you are just creating deps information with `-c`